packages/api/src/command/webhook/tenovi.ts

```diff
@@ -45,9 +46,9 @@ export const processMeasurementData = async (data: TenoviMeasurement): Promise<v
     const userData = mapData(data);
     createAndSendPayload(connectedUser, userData);
   } catch (error) {
-    console.log(`Failed to process Tenovi WH - error: ${JSON.stringify(error)}`);
+    console.log(`Failed to process Tenovi WH - error: ${String(error)}`);
     capture.error(error, {
-      extra: { context: `webhook.processMeasurementData`, error },
+      extra: { context: `webhook.processMeasurementData`, error, data: stringify(data) },
```

Comment:
no need to stringify, `capture.error` and `capture.message` already do this to stuff on `extra` 🤩

Reply:
Oh, ill keep that in mind! Ty